watertap/property_models/activated_sludge/asm1_properties.py

```diff
@@ -337,6 +337,11 @@ def energy_density_expression(self):
             rule=energy_density_expression, doc="Energy density term"
         )
 
+        iscale.set_scaling_factor(self.flow_vol, 1e1)
+        iscale.set_scaling_factor(self.temperature, 1e-1)
+        iscale.set_scaling_factor(self.pressure, 1e-3)
+        iscale.set_scaling_factor(self.conc_mass_comp, 1e1)
```

Comment:
We can (and probably should) set these at the flowsheet level

Reply:
ok

Reply:
@bknueven they should also be thrown in calculate_scaling_factors here, i.e., in the case that no scaling factor was provided by the user, set these.